buildScript/tasks.gincl

```diff
@@ -119,14 +119,15 @@ task createVersionTag  {
         props.setProperty('BuildMajor', major)
         props.setProperty('BuildMinor', minor)
         props.setProperty('BuildRev', rev)
-        props.setProperty('ClientBuildEnv', project.hasProperty("env") ? project.env : "local")
         props.setProperty('BuildType', type)
         props.setProperty('BuildNumber', buildNum)
```

Comment:
I think it would still be nice to keep access to the build environment.

Reply:
Build env is currently global:
https://github.com/Caltech-IPAC/firefly/blob/585c2a2704de025e134d1dffc9f454cb6a473d23/buildScript/webpack.config.js#L65

Let me know if you would like me to move it into our global `__PROPS__` instead.

Reply:
I did not realize that you were putting in in `NODE_ENV`. I thought `NODE_ENV` was suppose to have only two values `production` or `development`. 
I guess webpack and node only do code such as 
```js
NODE_ENV !== "production"
```
If it works that is fine.

Yes, I think it should be in __PROPS__ since that is how the client gets at it from the inside.